src/inventory/gilded_rose.js

```diff
@@ -21,59 +21,99 @@ const items = [
 
 updateQuality(items);
 */
+function checkNotSpecialItem(item) {
```

Comment:
Very minor nitpick but theres a convention that anything returning a boolean should begin with "is". This is less of a convention but I believe it helps with readability and that is that I avoid to use negatives in naming. In this case a function name could be `isOrdinaryItem`.  A link to an article of just OPINIONS but I think some are helpful https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig